sync/v1.proto

```diff
@@ -863,6 +875,36 @@ message Rule {
   ];
 }
 
+message FileAccessWatchItem {
+  enum RuleType {
+    RULE_TYPE_UNSPECIFIED = 0;
+    RULE_TYPE_PATHS_WITH_ALLOWED_PROCESSES = 1;
+    RULE_TYPE_PATHS_WITH_DENIED_PROCESSES = 2;
+    RULE_TYPE_PROCESSES_WITH_ALLOWED_PATHS = 3;
+    RULE_TYPE_PROCESSES_WITH_DENIED_PATHS = 4;
+  }
+  message Process {
+    oneof identifier {
+      string binary_path = 1;
+      string certificate_sha256 = 2;
+      string signing_id = 3;
+      string team_id = 4;
+      string cd_hash = 5;
+    }
+  }
+  string name = 1;
+  repeated string paths = 2;
```

Comment:
This will need to be a new type. Paths can be literal or prefix.

Reply:
Done